docs/src/Actions/Start.md

```diff
@@ -0,0 +1,78 @@
+# Redirect User to Frequency Access
+
+To start the authentication loop, your application will generate a redirect URL for each user.
+
+## Quick Reference
+
+- Staging-Testnet: `https://testnet.frequencyaccess.com/siwa/login`
+- Production-Mainnet: `https://www.frequencyaccess.com/siwa/login`
+- Generate Signed Request Payload: [`TODO`](#todo)
+- Request Structure: [`SiwaRequest`](../DataStructures/All.md#request)
+- Signed Request Structure: [`SiwaSignedRequest`](../DataStructures/All.md#signed-request)
+
+## Step 1: Generate the Signed Request Payload
+
+You *can* generate more than one of these for use simultaneously.
+Most applications should generate and keep their needed value(s) static.
+The generated signed request payload is secured by the signature included in it, and it is not sensitive data.
+
+### Signed Request Payload Contents
+
+- A signature from one of the [Control Keys](https://docs.frequency.xyz/Identity/ControlKeys.html) of the Frequency Provider
+- The domain and path the callback will use
+- The requested permissions
+- The requested credentials
+
+### Signed Request Payload Generator
+
+TODO: Link or embed this here
+
+### Signed Request Payload Detailed Documentation
+
+For details on the Payload and how it is formed, see the [Reference Documentation](../SignatureGeneration.md).
+
+## Step 2: Build the Redirect URL Parameters
+
+### Parameter: `signedRequest`
+
+The Base64 encoded value from Step 1.
+
+Reminder: This includes the callback domain that will be used.
+
+### Parameter: `callbackUrlParams`
+
+Additional unsigned parameters that will be appended to the callback URL.
+
+
+## Step 3: Build the Redirect URL
+
+Taking the parameters from the previous step, build a Redirect URL using the correct base:
```

Comment:
I really don't like calling it a redirect as it's the initial request into the flow, it's not a 3xx at all. That's like persnickety but every time I read redirect I assume I'm in the middle of something and being redirected somewhere else.